qemu/tests/cfg/block_performance_test.cfg

```diff
@@ -0,0 +1,35 @@
+- block_performance_test:
+    type = block_performance_test
+    only virtio_blk, virtio_scsi
+    images += " stg0 stg1"
+    force_create_image_stg0 = yes
+    force_remove_image_stg0 = yes
+    image_size_stg0 = 10G
+    image_name_stg0 = stg0
+    force_create_image_stg1 = yes
+    force_remove_image_stg1 = yes
+    image_size_stg1 = 11G
+    image_name_stg1 = stg1
+    kill_vm = yes
+    dd_cmd_timeout = 180
+    Linux:
+        dd_cmd = "time dd if=/dev/zero of=/dev/%s bs=1M count=5000 oflag=direct"
+    Windows:
+        dd_cmd = 'WIN_UTILS:\coreutils\DummyCMD.exe %s 1024000000 1'
+    variants:
+        - queue_size_test:
+            virtio_blk:
+                blk_extra_params_stg0 = "queue-size=128"
```

Comment:
set cache mode to none for data disk

Reply:
Thanks for your guidance, set some limits and retest this patch, but the performance is still not obvious.